app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -101,6 +107,9 @@ def alternate_machine_id(machine_id)
         # providers store it in downcase, so, we let the upcase/downcase logic to other methods with more
         # business knowledge.
         # 1 - https://bugzilla.redhat.com/show_bug.cgi?id=1294461
+        return if machine_id[0, 8].nil? || machine_id[8, 4].nil? || machine_id[12, 4].nil? || machine_id[16, 4].nil? ||
+                  machine_id[20, 12].nil?
```

Comment:
I don't think we an detect from *within* the container what the env outside is and/or in which OpenShift/K8s cluster it is running.